roles/ceph-osd-bcache/tasks/main.yml

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
Either way I'm fine with it. If we used one role would we want one group of hosts (`ceph_osds`)?